format/test_vertex_info.cc

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
What is the reason of that file?

Reply:
Sorry, this is the test file of POC, already removed it.